package.json

```diff
@@ -22,6 +22,7 @@
     "ink": "3.0.0-6",
     "nodegit": "^0.26.5",
     "prop-types": "^15.7.2",
+    "random-words": "^1.1.1",
```

Comment:
@saphal1998 could you check if this is the correct package? Your PR #31 uses "random-word" and idk which is the correct one haha